specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/expressRouteCircuit.json

```diff
@@ -2239,6 +2239,10 @@
         "globalReachEnabled": {
           "type": "boolean",
           "description": "Flag denoting global reach status."
+        },
+        "authorizationKey": {
```

Comment:
is this a secret?  If so it should not be in any responses and be marked as such with http://azure.github.io/autorest/extensions/#x-ms-mutability